client/src/components/Login/Login.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import './Login.js'
+import Card from 'react-bootstrap/Card';
+import Form from 'react-bootstrap/Form';
+import Container from 'react-bootstrap/Container';
+import Button from 'react-bootstrap/Button';
+import CardGroup from 'react-bootstrap/CardGroup';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+
+
+// FIXME: Review formating of extra columns
+export default function Login() {
+    const [validated, setValidated] = useState(false);
+
+    const handleSubmit = (e) => {
+        const form = e.currentTarget;
+        if (form.checkValidity() === false) {
+            e.preventDefault();
+            e.stopPropagation();
+        }
+
+        setValidated(true);
+    }
+
+    return (
+        <Container fluid>
+            <Row>
+                <Col />
+                <Col>
+                    <CardGroup>
+                        <Card style={{ width: '18rem' }}>
+                            <Card.Body>
+                                <Form noValidate validated={validated} onSubmit={handleSubmit} data-testid='valid-test'>
+                                    <Form.Group className='mb-3'>
+                                        <Form.Label htmlFor='email'>Email Address</Form.Label>
+                                        <Form.Control type='email' placeholder='someone@example.com' maxLength={40} id='email' required data-testid='email-input' />
+                                        <Form.Control.Feedback type='invalid' data-testid='email-err-msg'>Please enter a valid email.</Form.Control.Feedback>
+                                    </Form.Group>
+                                    <Form.Group className='mb-3'>
+                                        <Form.Label htmlFor='password'>Password</Form.Label>
+                                        <Form.Control type='password' placeholder='Enter Password' minLength={8} maxLength={24} id='password' required data-testid='password-input' />
```

Comment:
Add FIXME comments around the parts of the form that have magic numbers so we can move them to a new location eventually